README.md

```diff
@@ -5,8 +5,10 @@
 [![deploy on_netlify](https://img.shields.io/badge/deploy-on_netlify-blue.svg)](https://reposense.netlify.com/)
 [![codecov.io](https://codecov.io/gh/reposense/RepoSense/branch/master/graphs/badge.svg?branch=master)](http://codecov.io/github/reposense/RepoSense?branch=master)
 
-RepoSense is a contribution analysis tool for Git repositories. It generates a static HTML report including contribution information for each author in the repository.
-The features of the report includes:
+RepoSense is a contribution analysis tool for Git repositories. It is particularly useful for those who want a chronological visualisation of the code contributions and commits made to a group of repositories. Specifically, educators can use RepoSense to track the work done in student projects, while team leads may use it to oversee their project teams.
```

Comment:
code contribution instead of code contributions?

Reply:
I think it should be "contributions" since it's plural, right?

Reply:
haha yep you're right

Reply:
i think we can include more group of users e.g. project managers.